src/encoded/static/components/item-pages/components/ItemHeader.js

```diff
@@ -8,6 +8,7 @@ import queryString from 'query-string';
 import { console, object, schemaTransforms, memoizedUrlParse } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
 import { LocalizedTime } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/LocalizedTime';
 import { FlexibleDescriptionBox } from '@hms-dbmi-bgm/shared-portal-components/es/components/ui/FlexibleDescriptionBox';
+import { EditableField, FieldSet } from '@hms-dbmi-bgm/shared-portal-components/es/components/forms/components/EditableField';
```

Comment:
Codacy found an issue: ['EditableField' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5793828)